problems/countNumbers.js

```diff
@@ -9,8 +9,10 @@
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    for(let i = 0; i < arr.length; i++)
+    
+    return Object.keys(arr)
```

Comment:
This will return an array of all the keys in an object. If we do this and give it an array we'll get back an array of all the indicies of an array and each index will appear as a string.